content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md

```diff
@@ -205,8 +205,7 @@ policy for the embedded template to "`Never`".
 
 ## Job Termination and Cleanup
 
-When a Job completes, no more Pods are created, but the Pods are not deleted either.  Since they are terminated,
-they don't show up with `kubectl get pods`, but they will show up with `kubectl get pods -a`.  Keeping them around
+When a Job completes, no more Pods are created, but the Pods are not deleted either.  Keeping them around
```

Comment:
If you can add some words to describe the pod status, that will be much better.

Reply:
When a Job completes， it may succeed or fail. So we are not sure about pod status here. How to describe them?

Reply:
What does the output of `kubectl get pods` look like if a Job succeeds? Would it be useful to provide it, and then explain in a note what the Status column looks like for other cases?

Reply:
In the previous versions, I remember the pod status is `completed`. Maybe it differs now :)
I agree that we can describe its successful pod status here. That's my opinion. :)

Reply:
> What does the output of kubectl get pods look like if a Job succeeds? Would it be useful to provide it, and then explain in a note what the Status column looks like for other cases?

We can't directly distinguish whether a job is succeed/failed/completed from the output of `kubectl get pods`.

Then how to check whether a job is completed?
We need to check the status of a job.

Reply:
Changed ***When a Job completes*** to ***When status of a Job is Complete or Failed***